src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -3201,7 +3201,7 @@ public override void Visit(ParentNode node)
 
                 if (_nameResolver == null || _nameResolver.CurrentEntity == null)
                 {
-                    _txb.ErrorContainer.Error(node, TexlStrings.ErrInvalidName);
+                    _txb.ErrorContainer.Error(node, TexlStrings.ErrInvalidName, node.Id);
```

Comment:
>node.Id [](http://example.com/codeflow?start=80&length=7)

node.Id is an integer id for indexing into binder info; not a string to show the user. 
The tests added don't cover this case (I'm not sure how we'd actually hit this). 

We may need 2 separate errors, and to leave these cases unchanged.

Reply:
I put in the old error as "ErrInvalidIdentifier", and changed these to use the old error again and leave them basically unchanged. Thoughts?